package.json

```diff
@@ -8,7 +8,11 @@
     "lint:eslint": "yarn eslint . --ext ts,js",
     "lint:config-validation": "node ./scripts/validate-configs.js",
     "lint": "yarn lint:eslint && yarn lint:config-validation",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:config-validation --write"
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:config-validation --write",
+    "link-packages": "./scripts/link-packages.sh",
+    "create-release-pr": "./scripts/create-release-pr.sh",
+    "prepublishOnly": "yarn lint",
+    "publish-all": "lerna publish from-package"
```

Comment:
Does this also create a git tag? Does it push the tag?

Reply:
In my experience working with these scripts in the `snaps-skunkworks` repository, it does not create a tag, nor run any discernible `git` command. I think I've run it both when `HEAD` is already version-tagged and when it is not.

Reply:
Hmm. Well that's less than ideal. We should ensure that tags have been created and pushed by the time the publish to npm happens.

Reply:
To be totally honest, the primary benefit I hope to get out of release automation is the assurance that tags exist and are correct. I don't care nearly as much about the time saved, which often isn't significant. Whereas untagged releases have cost me a great deal of time personally, as I have to search through the commit history and compare the build output with the release to discover what the true tag is (as it's often not the commit where the version bump was done).

At the very least we should document in the release steps that git tags should be created and pushed - that's important enough to be a blocker I think. Automating it would be nice too but I'm OK with it being manual for now, since that's what it is now.